### ISSUE TITLE
Follower Service documentation or example?

### ISSUE THREAD

**Issue by germancryo** [TwitchLib/TwitchLib#1109]
Good day everyone,

I am looking for the documentation or an example to understand how to implement the Follower Service.
There is already a post about this, unfortunately the links to the example are no longer active.

Thanks in advance!

**Comment by RickyDivjakovski** [comment]
Agree, theres documentation on getting s follower list, maybe push it through to an array and have a timer refresh that list looking for changes every 60 seconds

Just throwing some options out there until the devs get back to you, im currently facing the same problem, theres no OnFollower event either
